### DISCUSSION TITLE
Embeddable insights view

### DISCUSSION THREAD

**Discussion by claytongulick** [directus/directus#15774] [Feature Requests]
The insights functionality is great, but would be more useful if it could be embedded into an application via iframe.

There's a fullscreen toggle button now, if there were a URL param (may there is one and I can't find it? I've looked in docs and source code) that was something like full_screen=true, it would allow us to embed the insights view into our UI application. It should pick up the existing directus auth cookies because we're using the js sdk - so would "just work".

We want to avoid having clients log into the directus admin UI, but as the insights functionality continues to improve having an embeddable dashboard view would make it a feature we'd use heavily.

**Comment by rijkvanzanten** [comment]
Love that idea! I'm wondering if we could reuse the Share functionality we currently have in the content module item detail pages, but on insights dashboards. That way, you could generate a semi-private link that you can share with other people to just see that particular dashboard, and could subsequently also use that link in an iframe in other platforms 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by bastien-g** [comment]
This would be a key feature for me
